DAQ_System/Source/Adapter/DataLogger/SdDataLogger.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+* SD Data Logger
+* Authors:        Cristian Cruz
+*                 Ethan Fischer
+*
+* Email:          cris14.alex@gmail.com
+*                 ejf180001@utdallas.edu
+* 
+* (c) 2022 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#include "SdDataLogger.hpp"
+#include "FATFileSystem.h"
+#include "SDBlockDevice.h"
+
+namespace adapter {
```

Comment:
Thoughts on moving this to the Application layer?

Reply:
Done